modules/nf-core/ensemblvep/vep/meta.yml

```diff
@@ -20,29 +23,45 @@ input:
       type: file
       description: |
         vcf to annotate
+  - custom_extra_files:
+      type: file
+      description: |
+        extra sample-specific files to be used with the `-custom` flag
+        => add this yourself in the `args`
+        (optional)
   - genome:
-      type: value
+      type: string
       description: |
         which genome to annotate with
   - species:
-      type: value
+      type: string
       description: |
         which species to annotate with
   - cache_version:
-      type: value
+      type: integer
       description: |
         which version of the cache to annotate with
+  - meta2:
+      type: map
+      description: |
+        Groovy Map containing cache information
+        e.g. [ id:'test' ]
   - cache:
       type: file
       description: |
         path to VEP cache (optional)
+  - meta3:
+      type: map
+      description: |
+        Groovy Map containing fasta reference information
+        e.g. [ id:'test' ]
   - fasta:
       type: file
       description: |
         reference FASTA file (optional)
       pattern: "*.{fasta,fa}"
   - extra_files:
-      type: tuple
+      type: map
```

Comment:
is this really a map and not tuple of files? (like an array or something)

Reply:
Yes, it is but the linter is complaining that `tuple` is not a valid type (and `map` was the closest thing it accepts 😁)

Reply:
list?

Reply:
I'd rather have `file` than `map` as I feel `map` is not really the right type at all

Reply:
I agree!